src/Parser/EnhancementShaman/Modules/ShamanCore/Flametongue.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class Flametongue extends Module {
+
+  suggestions(when) {
+    const flametongueUptime = this.owner.selectedCombatant.getBuffUptime(SPELLS.FLAMETONGUE_BUFF.id) / this.owner.fightDuration;
+    when(flametongueUptime).isLessThan(.95)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(`Your Flametongue uptime of ${formatPercentage(flametongueUptime)}% is below 95%, try to get as close to 100% as possible`)
+          .icon(SPELLS.FLAMETONGUE_BUFF.icon)
+          .actual(`${formatPercentage(flametongueUptime)}% uptime`)
```

Comment:
This should use the `actual` value instead, so you don't have to worry about keeping both values in sync when the threshold changes.